packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -160,8 +163,23 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
     hasError,
   });
 
+  const contextValue: ControlsContext = {
+    data: {
+      pagination: {
+        currentPage,
+        disableNext,
+        disablePrevious,
+        handlePaginateNext: () =>
+          handlePaginateNext({ resultCount, hasNextToken }),
+        handlePaginatePrevious: () =>
+          handlePaginatePrevious({ resultCount, hasNextToken }),
+      },
+    },
+    actionsConfig: { type: 'LIST_LOCATIONS', isCancelable: false },
```

Comment:
Need to understand this, please explain

Reply:
I initially misunderstood what `actionsConfig` was there for and why it was required. Sorry, I meant to include a question on this line before posting for review but I forgot to.

Would `SINGLE_ACTION` be the appropriate type of action for pagination? Or is there something I'm missing?